extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -360,6 +370,40 @@ describes.sandboxed('amp-ad-network-adsense-impl', {}, () => {
         expect(ampAnalyticsElement.innerHTML.indexOf(url)).to.not.equal(-1);
       });
     });
+    it('centers iframe in slot when height && width', () => {
+      const centerCreativeSpy = sandbox.spy(impl, 'centerCreative_');
+      impl.onCreativeRender(false);
+      expect(centerCreativeSpy).to.be.calledOnce;
+    });
+    it('centers iframe in slot when !height && !width', () => {
+      return createImplTag({
```

Comment:
Yes, the test is running against the correct instance. I've added expectations to the tests to make this explicit.